package.json

```diff
@@ -17,6 +17,7 @@
     "clean": "lerna run clean",
     "build": "lerna run build",
     "test": "yarn workspace foam-vscode test --stream",
+    "test:web": "yarn workspace foam-vscode test:web",
```

Comment:
I think the top level `test` should call both test targets on the package, so we have a consistent API at this level

Reply:
Would that also mean that you don't need the extra step in the CI? I am not totally opposed to it (we could call the task on the package if anything)